src/db.js

```diff
@@ -26,7 +26,7 @@ const sequelize = new Sequelize({
   host: POSTGRES_HOST || 'localhost',
   dialect: 'postgres',
   port: POSTGRES_PORT || 5432,
-  password: POSTGRES_PASSWORD || '123',
+  password: POSTGRES_PASSWORD || '123123',
```

Comment:
Avoid hardcoding sensitive information like passwords in the codebase. Consider using environment variables or a secure vault to manage credentials.